donkeycar/parts/camera.py

```diff
@@ -329,6 +331,61 @@ def shutdown(self):
         self.running = False
         time.sleep(0.5)
 
+class ScreenCamera(BaseCamera):
+    '''
+    Camera that uses mss to capture the screen
+    For capturing video games
+    '''
+
+    def __init__(self, image_w=160, image_h=120, image_d=3,
+                 vflip=False, hflip=False):
+        self.image_w = image_w
+        self.image_h = image_h
+        self.image_d = image_d
+        self.vflip = vflip
+        self.hflip = hflip
+        self.sct = mss.mss()
+        self.running = True
+
+        self._monitor_thread = threading.Thread(target=self.take_screenshot, args=())
```

Comment:
Please have a look at other threaded parts. Threads are not started in the constructor but centrally in the car application, when adding the part as a threaded part - which is what you want to do for your part.